go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	golang.org/x/net v0.0.0-20200904194848-62affa334b73
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208 // indirect
 	golang.org/x/sys v0.0.0-20200905004654-be1d3432aa8f
+	golang.org/x/tools v0.0.0-20200423205358-59e73619c742 // indirect
```

Comment:
I'm not sure about but this dep can go to buildtool installation script since it is not needed for building and running algod.

Reply:
agreed, removed.